src/foam/core/ClientRuntimeException.js

```diff
@@ -11,6 +11,16 @@ foam.CLASS({
   extends: 'foam.core.FOAMException',
   javaGenerateConvenienceConstructor: false,
 
+  properties: [
+    {
+      name: 'cause_',
```

Comment:
Our foam.core.FOAMException should have added the getClientRethrowException. This is messy, but don't see a way out now.